e2e-tests/components/toolbar/components/copy-paste/utils.js

```
@@ -27,50 +27,50 @@
+		'box-shadow-palette-status-xl-hover': true,
 		'container-size-advanced-options': false,
-		'container-min-width-unit-general': 'px',
-		'container-max-height-unit-general': 'px',
-		'container-height-unit-general': 'px',
-		'container-min-height-unit-general': 'px',
-		'container-max-width-general': 1170,
+		'container-min-width-unit-xl': 'px',
+		'container-max-height-unit-xl': 'px',
+		'container-height-unit-xl': 'px',
+		'container-min-height-unit-xl': 'px',
+		'container-max-width-xl': 1170,
```

Biome (1.9.4)

[error] This property value named container-max-width-xl is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named container-max-width-xl


(lint/suspicious/noDuplicateObjectKeys)